packages/plugin-eslint/src/lib/eslint-plugin.integration.test.ts

```diff
@@ -18,10 +19,19 @@ describe('eslintPlugin', () => {
     ...plugin,
     runner: {
       ...(plugin.runner as RunnerConfig),
-      args: (plugin.runner as RunnerConfig).args?.map(arg =>
-        toUnixPath(arg.replace(path.dirname(thisDir), '<dirname>')),
-      ),
-      outputFile: toUnixPath((plugin.runner as RunnerConfig).outputFile),
+      args: (plugin.runner as RunnerConfig).args?.map(arg => {
+        if (arg.includes('bin.js')) {
+          return toUnixPath(arg.replace(path.dirname(thisDir), '<dirname>'));
+        }
+
+        return toUnixPath(arg).replace(
+          /\/eslint\/\d+\//,
+          '/eslint/<timestamp>/',
+        );
+      }),
```

Comment:
This can be simplified:

```suggestion
      args: (plugin.runner as RunnerConfig).args?.map(arg =>
        toUnixPath(arg.replace(path.dirname(thisDir), '<dirname>')).replace(
          /\/eslint\/\d+\//,
          '/eslint/<timestamp>/',
        ),
      ),
```